index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+thebroadwalk.com
+        </title>
+  <link rel="stylesheet" href="eventhall.css">
+    </head>
+    <body>
+<header></header> <img src="header.png"> alt="header image" </header>
+<div class="topnav">
+ <a class="active" href="#Home">Home</a>
+    <a href="contact">Contact</a>
+    <a href="about">About</a>
+    <p><strong>THE BOARDWALK MARCH PROMO</strong></p>
+Step into spring with 40% OFF ALL our spaces on ANY date in the year AND to make this deal smell even sweeter UNINTERRUPED POWER (valued at over N100,000 for use with booking of our Courtyard and Hollywood Hall Only) for your event.
+<p></p>
+<p></p>
+<p>Hurry and BOOK NOW!</p>
+<p></p>
+<p>Dates are selling fast!</p>
+
+<p>Promo ends on the 31st of March 2020.</p>
+<p></p>
+<p><b> <p>THE HOLLYWOOD HALL (150 banquet, 200 theatre)</p>
+Retail price – N750,000
+March sale price – N450,000</p>
+</p>
+<p>THE SUNSET ROOM (40 banquet, 60 theatre)
```

Comment:
The page should be well-styled generally to reflect margins, font-sizes, color, padding etc.